docs/basico/store.md

```diff
@@ -73,4 +73,4 @@ let store = createStore(todoApp)
 
 ## Próximos Pasos
 
-Antes de crear la interfaz de usuario para nuestra aplicación TODO, tomaremos un desvío para ver [cómo fluyen los datos en una aplicación Redux](flujodata.md).
+Antes de crear la interfaz de usuario para nuestra aplicación TODO, tomaremos un desvío para ver [cómo fluyen los datos en una aplicación Redux](flujo-de-datos.md).
```

Comment:
El archivo se llama [flujos-de-datos.md](https://github.com/sergiodxa/redux-in-spanish/blob/master/docs/basico/flujo-de-datos.md), la ruta es correcta